mantle/kola/tests/misc/network.go

```diff
@@ -490,6 +563,20 @@ func checkExpectedMAC(c cluster.TestCluster, m platform.Machine, interfaceName,
 	}
 }
 
+func checkConnectionsUp(c cluster.TestCluster, m platform.Machine, connectionNames []string) error {
+	failedConnections := []string{}
+	for _, connection := range connectionNames {
+		activeConnectionGrep := string(c.MustSSH(m, fmt.Sprintf("nmcli -g NAME con show --active | grep '%s' || true", connection)))
```

Comment:
This is not as tight as the previous approach. E.g. connection `foo` would also match `foobar`. I think you could just split the output of `nmcli -g NAME con show --active` and we have a function I think that checks if a string is in a slice of strings.

But cool too to just roll with this for now if you'd like.

Reply:
you're right. I've added `grep -w`, and used a function `grepActiveConnections` to assist for this issue. [Done](https://github.com/coreos/coreos-assembler/compare/a91f506ae1238b32de32e9f847335b40a164dce7..88399a75d96ad667571a1701b71ad445635e0e6b)

Reply:
Hmm, but `grep -w` doesn't help with e.g. `foo bar`, no? Wouldn't it be simpler and cleaner to skip using `grep` entirely and gather all the active connection names and split on newline?

Reply:
I totally agree, removed the grep entirely :) Thanks!